requirements.txt

```diff
@@ -5,4 +5,4 @@ numpy>=1.16.4
 torch>=1.1
 tensorboard>=1.14
 future>=0.17.1  # required for builtins in setup.py
-
+pyyaml>=5.3.1
```

Comment:
is it really the minimal version we can use?

Reply:
it's the current latest version

Reply:
Could we put there like 3.x or lover since we are using quite general features?

Reply:
@S-aiueo32 Don't forget to also add it to the environment.yml file